frontend/src/styles/_uswds-theme-custom-styles.scss

```diff
@@ -493,3 +503,9 @@ button.usa-pagination__button.usa-button {
     overflow-wrap: break-word;
   }
 }
+
+// Accordions
+// .usa-accordion {}
+.usa-accordion__heading {
+  border-bottom: 2px solid color(gray-warm-30);
+}
```

Comment:
Accordion heading buttons are the same color as the alt section backgrounds. This border makes them more apparent. (see [Slack thread](https://betagrantsgov.slack.com/archives/C05TGEL3C6Q/p1747850416379299?thread_ts=1747844913.256969&cid=C05TGEL3C6Q))